yamls/ndlar_flow/reco/charge/CalibHitBuilderMC.yaml

```diff
@@ -26,6 +26,8 @@ params:
   calib_hits_dset_name: 'charge/calib_prompt_hits'
   mc_hit_frac_dset_name: 'mc_truth/calib_prompt_hit_backtrack'
 
+  hit_ref: False
```

Comment:
Default to `True` so the production included hit backtracking references

Reply:
It is one to one correspondence by order from calib_prompt hits and backtracking. (Same as in larndsim.) This reference is built after the affect which I think is not necessary. We might need to impose a check on the backtracking size and hits size.

Reply:
Sure, but it can change moving forward and hit order can be changed. I prefer robust solutions to ones that require unspoken conventions to be followed.